ui/components/app/snaps/keyring-snap-removal-warning/keyring-snap-removal-warning.tsx

```diff
@@ -91,46 +91,48 @@ export default function KeyringRemovalSnapWarning({
           >
             {t('removeSnap')}
           </ModalHeader>
-          {showConfirmation === false ? (
-            <>
-              <BannerAlert severity={BannerAlertSeverity.Warning} className="">
-                {t('backupKeyringSnapReminder')}
-              </BannerAlert>
-              <Box
-                display={Display.Flex}
-                justifyContent={JustifyContent.spaceBetween}
-              >
-                <Text>{t('removeKeyringSnap')}</Text>
-                <InfoTooltip
-                  contentText={t('removeKeyringSnapToolTip')}
-                  position="top"
-                />
-              </Box>
-              {keyringAccounts.map((account, index) => {
-                return (
-                  <KeyringAccountListItem
-                    key={index}
-                    account={account}
-                    snapUrl={getAccountLink(account.address, chainId)}
+          <ModalBody
+            display={Display.Flex}
+            flexDirection={FlexDirection.Column}
+            gap={4}
+          >
+            {showConfirmation === false ? (
+              <>
+                <BannerAlert
+                  severity={BannerAlertSeverity.Warning}
+                  className=""
+                >
+                  {t('backupKeyringSnapReminder')}
+                </BannerAlert>
+                <Box
+                  display={Display.Flex}
+                  justifyContent={JustifyContent.spaceBetween}
+                >
+                  <Text>{t('removeKeyringSnap')}</Text>
+                  <InfoTooltip
+                    contentText={t('removeKeyringSnapToolTip')}
+                    position="top"
```

Comment:
Refactoring to wrap the content of this modal in `ModalBody` which handles the scrolling of the modal